draft-ietf-rats-corim.md

```diff
@@ -111,59 +109,117 @@ informative:
+Reference values and Endorsements are required for Verifier reconciliation (and Evidence is required for corresponding internal ACS creation) as illustrated in {{sec-interact-acs}}.
+
+## Internal Representation {#sec-internal-rep}
+
+In this document CDDL is used to specify both the CoRIM structure and to specify an internal representation for use in the appraisal procedure.
+The actual internal representation of a Verifier is implementation-specific and out-of-scope of this document.
+Requirements for an internal representation of Conceptual Messages are defined in {{tbl-cmrr}}, where each Conceptual Message type has a structure as depicted by the *Structure* column.
+The internal representations used by this document are defined in {{sec-ir-cm}}.
+
+## Interacting with an ACS {#sec-interact-acs}
+
+Conceptual Messages interact with an ACS by specifying criteria that should be met by the ACS and by presenting the assertions that should be added to the ACS if the criteria are satisfied.
+Internal representations of Conceptual Messages, ACS, and ARS should satisfy the following requirements for Verifier reconciliation and appraisal processing:
+
+| CM Type | Structure | Description |
+|---
+| Evidence | List of Evidence claims | If the Attester is authenticated, add Evidence claims to the ACS with Attester authority |
+| Reference Values | List of Reference Values claims | If a reference value in a CoRIM matches claims in the ACS, then the authority of the CoRIM issuer is added to those claims. |
+| Endorsements | List of expected actual state claims, List of Endorsed Values claims | If the list of expected claims are in the ACS, then add the list of Endorsed Values claims to the ACS with Endorser authority |
+| Series Endorsements | List of expected actual state claims, Series of tuples containing selection and series Endorsement claims | If the list of expected claims are in the ACS, and if the series selection condition is satisfied, then add the series Endorsement claims for the first series tuple with satisfied condition to the ACS with Endorser authority |
+| Verifier | List of expected actual state claims, List of Verifier-generated claims | If the list of expected claims are in the ACS, then add the list of Verifier-generated claims to the ACS with Verifier authority |
+| Policy | List of expected actual state claims, List of Policy-generated claims | If the list of expected claims are in the ACS, then add the list of Policy-generated claims to the ACS with Policy Owner authority |
+| Attestation Results | List of expected actual state claims, List of expected Attestation Results claims | If the list of expected claims are in the ACS, then copy the list of Attestation Results claims into the ARS |
+{: #tbl-cmrr title="Conceptual Message Representation Requirements"}
+
+## Quantizing Inputs {#sec-quantize}
+
+To be completed!
```

Comment:
add a reference to the tracking issue instead

Reply:
issue #242